webapp/components/features/FollowList/FollowList.vue

```diff
@@ -0,0 +1,165 @@
+<template>
+  <base-card class="follow-list">
+    <template v-if="connections && connections.length">
+      <h5 class="title spacer-x-small">
+        {{ userName | truncate(15) }} {{ $t(`profile.network.${type}`) }}
+      </h5>
+      <ul :class="connectionsClass">
+        <li
+          v-for="connection in filteredConnections"
+          :key="connection.id"
+          class="connections__item"
+        >
+          <user-teaser :user="connection" />
+        </li>
+      </ul>
+      <base-button
```

Comment:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2020-04-07T15:01:03Z" title="Tuesday, April 7th 2020, 5:01:03 pm +02:00">Apr 7, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,150 @@
+<template>
+  <base-card class="follow-list">
+    <template v-if="connections && connections.length">
+      <h5 class="title spacer-x-small">
+        {{ userName | truncate(15) }} {{ $t(`profile.network.${type}`) }}
+      </h5>
+      <div :class="connectionsClass">
+        <user-teaser
+          v-for="connection in filteredConnections"
+          :key="connection.id"
+          :user="connection"
+          class="spacer-x-small"
+        />
+      </div>
+      <base-button
+        v-if="allConnectionsCount - connections.length"
```


So `0` is `falsy`? Is this maybe more readable:
```suggestion
        v-if="allConnectionsCount > connections.length"
```

Reply:
> [<img alt="rbeer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/rbeer) **Authored by [rbeer](https://github.com/rbeer)**
_<time datetime="2020-04-09T21:02:43Z" title="Thursday, April 9th 2020, 11:02:43 pm +02:00">Apr 9, 2020</time>_

---


0 and '' are falsy in JS, yes. 
>The profile page had the count of displayed connections hardcoded to 7; I just took that over, when I started.
It makes way more sense now to introduce a `hasMore` computed value of `user.followingCount > user.following.length`

Reply:
> [<img alt="rbeer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/rbeer) **Authored by [rbeer](https://github.com/rbeer)**
_<time datetime="2020-04-09T21:55:31Z" title="Thursday, April 9th 2020, 11:55:31 pm +02:00">Apr 9, 2020</time>_

---


e018e2ead